### DISCUSSION TITLE
I can't open localsend

### DISCUSSION THREAD

**Discussion by AndREw1212hahaha** [localsend/localsend#1645] [Discussion (English, International)]
Yesterday I installed localsend on my computer (windows 10) (it still works), windows 10 updates automatically when I shut it down, today when I double-click localsend on my computer, it shows open in the taskbar, but the localsend home page is not displayed on the home screen. To see if the task manager is in the allowed state, I reinstalled localsend and it didn't work. I hope we can solve this         problem.   https://github.com/user-attachments/assets/ba96fcd6-9f2f-4ff7-8e53-d44ec675a768

**Comment by SShwinA** [comment]
Does the portable version work?

**Comment by DrOncogene** [comment]
I have a similar problem. I installed localsend, initially via winget and then using the .exe installer. It refuses to open each time. I use a windows 11 machine
